keyboards/ai03/orbit_x/info.json

```diff
@@ -0,0 +1,61 @@
+{
+    "keyboard_name": "Orbit-X", 
+    "url": "", 
+    "maintainer": "ai03", 
+    "width": 13.5, 
+    "height": 4.38, 
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"x":3, "y":0},
+                {"x":9.5, "y":0},
+                {"x":2, "y":0.13},
+                {"x":4, "y":0.13},
+                {"x":8.5, "y":0.13},
+                {"x":10.5, "y":0.13},
+                {"x":5, "y":0.25},
+                {"x":7.5, "y":0.25},
+                {"x":0, "y":0.38},
+                {"x":1, "y":0.38},
+                {"x":11.5, "y":0.38},
+                {"x":12.5, "y":0.38},
+                {"x":3, "y":1.0},
+                {"x":9.5, "y":1.0},
+                {"x":2, "y":1.13},
+                {"x":4, "y":1.13},
+                {"x":8.5, "y":1.13},
+                {"x":10.5, "y":1.13},
+                {"x":5, "y":1.25},
+                {"x":7.5, "y":1.25},
+                {"x":0, "y":1.38},
+                {"x":1, "y":1.38},
+                {"x":11.5, "y":1.38},
+                {"x":12.5, "y":1.38},
+                {"x":3, "y":2},
+                {"x":9.5, "y":2},
+                {"x":2, "y":2.13},
+                {"x":4, "y":2.13},
+                {"x":8.5, "y":2.13},
+                {"x":10.5, "y":2.13},
+                {"x":5, "y":2.25},
+                {"x":7.5, "y":2.25},
+                {"x":0, "y":2.38},
+                {"x":1, "y":2.38},
+                {"x":11.5, "y":2.38},
+                {"x":12.5, "y":2.38},
+                {"x":3, "y":3},
+                {"x":9.5, "y":3},
+                {"x":2, "y":3.13},
+                {"x":10.5, "y":3.13},
+                {"x":5.25, "y":3.25, "h":1.5},
+                {"x":7.25, "y":3.25, "h":1.5},
+                {"x":0, "y":3.38},
+                {"x":1, "y":3.38},
+                {"x":4.25, "y":3.38},
+                {"x":8.25, "y":3.38},
+                {"x":11.5, "y":3.38},
+                {"x":12.5, "y":3.38}
```

Comment:
The KLE to info.json Converter doesn't play nice with keyboards that run column stagger.

```suggestion
                {"x":0, "y":0.38},
                {"x":1, "y":0.38},
                {"x":2, "y":0.13},
                {"x":3, "y":0},
                {"x":4, "y":0.13},
                {"x":5, "y":0.25},
                {"x":7.5, "y":0.25},
                {"x":8.5, "y":0.13},
                {"x":9.5, "y":0},
                {"x":10.5, "y":0.13},
                {"x":11.5, "y":0.38},
                {"x":12.5, "y":0.38},
                {"x":0, "y":1.38},
                {"x":1, "y":1.38},
                {"x":2, "y":1.13},
                {"x":3, "y":1.0},
                {"x":4, "y":1.13},
                {"x":5, "y":1.25},
                {"x":7.5, "y":1.25},
                {"x":8.5, "y":1.13},
                {"x":9.5, "y":1.0},
                {"x":10.5, "y":1.13},
                {"x":11.5, "y":1.38},
                {"x":12.5, "y":1.38},
                {"x":0, "y":2.38},
                {"x":1, "y":2.38},
                {"x":2, "y":2.13},
                {"x":3, "y":2},
                {"x":4, "y":2.13},
                {"x":5, "y":2.25},
                {"x":7.5, "y":2.25},
                {"x":8.5, "y":2.13},
                {"x":9.5, "y":2},
                {"x":10.5, "y":2.13},
                {"x":11.5, "y":2.38},
                {"x":12.5, "y":2.38},
                {"x":0, "y":3.38},
                {"x":1, "y":3.38},
                {"x":2, "y":3.13},
                {"x":3, "y":3},
                {"x":4.25, "y":3.38},
                {"x":5.25, "y":3.25, "h":1.5},
                {"x":7.25, "y":3.25, "h":1.5},
                {"x":8.25, "y":3.38},
                {"x":9.5, "y":3},
                {"x":10.5, "y":3.13},
                {"x":11.5, "y":3.38},
                {"x":12.5, "y":3.38}
```